test/unit/app/models/reshuffle_mode_test.rb

```diff
@@ -0,0 +1,72 @@
+require "test_helper"
+
+class ReshuffleModeTest < ActiveSupport::TestCase
+  extend Minitest::Spec::DSL
+
+  class ClassThatIncludesReshuffleMode < ApplicationRecord
```

Comment:
If you don't want to bring the whole of ActiveRecord along for the ride, you can do something like this: https://github.com/alphagov/whitehall/blob/7abf9c0587c6818e06748835cebb1fc0ff5fbe3d/test/unit/app/models/user_orderable_test.rb#L4

Not sure if that works in this situation, it can be more awkward if you do need some ActiveRecord behaviours

Reply:
This didn't work - but it did result in me removing one layer of inheritance by inheriting from `ActiveRecord::Base` rather than `ApplicationRecord`.

Reply:
EDIT: not allowed!

```
test/unit/app/models/reshuffle_mode_test.rb:6:42: C: [Correctable] Rails/ApplicationRecord: Models should subclass ApplicationRecord.
  class ClassThatIncludesReshuffleMode < ActiveRecord::Base
```